File: jupyter2/problems/iteration/minimize_polynomial/go/minimize_polynomial.go
package main
import "math"
import "fmt"

/** begin my answer */

/** end my answer */

func main() {
	if !(math.Abs(minValue(-1, 1, 1000) - 0.903266) < 1.0e-6) { panic("wrong") }
	if !(math.Abs(minValue( 1, 2, 1000) - 1.928766) < 1.0e-6) { panic("wrong") }
	fmt.Println("OK")
}

File: jupyter2/problems/iteration/minimize_polynomial/ml/minimize_polynomial.ml
(** begin my answer *)

(** end my answer *)

let main () =
  assert ((abs_float (min_value (-1.0) 1.0 1000) -. 0.903266) < 1.0e-6);
  assert ((abs_float (min_value   1.0  2.0 1000) -. 1.928766) < 1.0e-6);
  Printf.printf "OK\n"
;;

main()
